pkg/config/setup/config.go

```diff
@@ -357,7 +357,10 @@ func InitConfig(config pkgconfigmodel.Setup) {
 
 	// flare configs
 	config.BindEnvAndSetDefault("flare_provider_timeout", 10*time.Second)
-	config.BindEnvAndSetDefault("flare.rc_profiling_runtime", 60*time.Second)
+	config.BindEnvAndSetDefault("flare.rc_profiling_runtime", 30*time.Second)
```

Comment:
Would it make sense to add these to `config_template.yaml`?

Reply:
Last pr conversation with shared-components suggested that undocumented options were fine for these, but I will revisit now that it's actually being used.